usr/share/lib/ipa/tests/SLES/Azure/test_sles_azure_accel_networking.py

```diff
@@ -0,0 +1,3 @@
+def test_sles_azure_accel_networking(host):
+    result = host.run('sudo lspci')
+    assert 'Mellanox' in result.stdout
```

Comment:
This should be:
`has_melanox_hw = 'Melanox' in host.run(sudo lspci).stdout
has_mlx_driver = 'mlx_' in host.run(sudo lsmod).stdout

if has_melanox_hw:
    assert has_mlx_driver`

Reply:
Updated, match is `mlx4_`.